config/settings.yml

```diff
@@ -242,6 +242,7 @@ modules_appeals_api:
     region: "region"
     bucket: "bucket"
   status_simulation_enabled: false
+  legacy_appeals_enabled: false
```

Comment:
There's a `config/settings/test.yml` that we can use to set this to `true` and remove all the `with_settings` additions.

Reply:
ok sweet. thanks